### ISSUE TITLE
Move requests between collections

### ISSUE THREAD

**Issue by nikischin** [usebruno/bruno#3320]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

[Jira](https://usebruno.atlassian.net/browse/BRU-1399)

### Describe the feature you want to add

I would like to move requests/folders between collections using the UI. Currently it is only possible to move requests/folders within a collection.

### Mockups or Images of the feature

![Mockup](https://github.com/user-attachments/assets/181194ce-a92e-46d1-b592-05600625e140)

**Comment by TheRealRyanReynolds** [comment]
Thanks @nikischin! Is this capability something you use today in Postman? Would love to hear a bit more about your use case.

**Comment by nikischin** [comment]
Ah no it has nothing to do with Postman. It's just some times you don't exactly know how to split your collections.

For me I wanted to create a collection per service and also a separate collection to cover all API test cases (including negative Tests). So because I needed all my sample data which is saved in local variables from one service in another one I decided to move both services in one Collection instead of two.

In future when I will have a separate collection which cover all tests including negative tests I will want to duplicate request within one collection in order to move it to the other one. I can do this via file system luckily, however it would be handy/intuitive doing it via the UI.

**Comment by TheRealRyanReynolds** [comment]
Thanks @nikischin , that's really helpful. I know moving collections like that is something available in other tools, which is why I asked. 

We'll need to add this to our planning. In the spirit of transparency, I'm not sure it's something we'll get to in 2024. Right now the team is heads-down on crossing off blocking issues. Quality of life enhancements are really important to us and we totally get it, but need a little bit of time before we'll be able to shift.

**Comment by nikischin** [comment]
@TheRealRyanReynolds yeah sure no worries! I just wanted it to be reported but it does not have the highest priority for me.

#2397 though would be a real game changer for me, as it would introduce many new opportunities of how to use bruno to test complex applications, so if this is something you could upvote, I would be really happy! 😃

**Comment by kbublitz** [comment]
Have my upvote. I just imported some collections and wanted to reorganize them, so the only way was to move the bru files in the file explorer. Luckily Bruno has this nice file based approach, so it's not a big deal doing so.
It is not a daily thing I do, but it does happen from time to time that I want to move (or mostly duplicate) requests to another collection.
I totally get that there are other issues that are more important.

**Comment by ElementalBear** [comment]
For me this feature became useful when I was attempting to import my Postman collections and then attempting to merge/reorganize them to one Bruno collection (merging to one collection since environments are per collection and I do not want to duplicate the data). Right now, I have to sit there and copy/paste hundreds of requests from the imported collection to my new Bruno collection that is better organized.

**Comment by jhoos70** [comment]
Pretty basic feature necessary when reorganizing collections.

**Comment by goodmuyis** [comment]
This has my upvote.
In the meantime as work around, you can copy and paste the `.bru ` file from the collection folder to another; it works without restarting Bruno.

**Comment by conradvc-kaha** [comment]
Any update on this? It would be hugely valuable to me as well.

**Comment by jayakrishnancn** [comment]
created a [PR](https://github.com/usebruno/bruno/pull/5410) for this feature.

**Comment by helloanoop** [comment]
PR has been merged. This sill be available in `v2.12.0` released scheduled to go out in last week of Sep 2025
